waspc/data/Generator/templates/server/src/dbClient.js

```diff
@@ -1,42 +1,17 @@
 {{={= =}=}}
 import Prisma from '@prisma/client'
-{=# isAuthEnabled =}
-import { hashPassword } from './core/auth.js'
-{=/ isAuthEnabled =}
 
-{=# isAuthEnabled =}
-const PASSWORD_FIELD = 'password'
+import { registerAuthMiddleware } from './core/auth/middleware.js'
```

Comment:
You might want to also put IsAuthEnabled check around this import, since we had it around ./core/auth.js before.
I wonder why we had it, is there more to it then avoiding redundant import -> maybe that file is not even created if auth is not enabled? Can't remember, but having this guards will certainly eliminate any issue.

Reply:
Will look into that, thanks!

Reply:
This is addressed here: https://github.com/wasp-lang/wasp/pull/376/commits/ba6479e6b0e346a7a75a608243c69de58e1ab908 Thanks!